src/cases/components/case-hearings-list/case-hearings-list.component.html

```diff
@@ -25,11 +25,12 @@
     </td>
     <td class="govuk-table__cell govuk-table-width-30-percent">
       <div *ngIf="hasUpdateAction" class="div-action">
-        <a href="javascript:void(0)" (click)="viewAndEdit()">View or edit hearing</a>&nbsp;&nbsp;&nbsp;&nbsp;
+        <ng-container *ngIf="!hasAddEdit(hearing)"><a href="javascript:void(0)" (click)="viewAndEdit()">View or edit</a>&nbsp;&nbsp;&nbsp;&nbsp;</ng-container>
```

Comment:
already done in https://github.com/hmcts/rpx-xui-webapp/pull/1773

Reply:
@paddybasi - this is now outdated as I merged the changes from base branch after you merged your changes (EUI-5092).